src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs

```diff
@@ -1,65 +1,73 @@
 {
-    public class TexUnderlineAnnotator : AbstractSvgAnnotator
+    public class TexUnderlineAnnotator : BaseAnnotator
     {
+        private UnderlineAnnotation underlineAnnotation;
+
         public TexUnderlineAnnotator(AnnotationDataEntity annotationData, PageData pageData)
             : base(annotationData, pageData)
         {
+            underlineAnnotation = new UnderlineAnnotation()
+            {
+                Points = new List<Point>
+                {
+                    new Point(annotationData.left, pageData.Height - annotationData.top),
+                    new Point(annotationData.left + annotationData.width, pageData.Height - annotationData.top),
+                    new Point(annotationData.left, pageData.Height - annotationData.top - annotationData.height),
+                    new Point(annotationData.left + annotationData.width, pageData.Height - annotationData.top - annotationData.height)
+                }
+            };
         }
-        
-        public override AnnotationInfo AnnotateWord()
+
+        public override AnnotationBase AnnotateWord()
         {
-            SetFixTop(true);
-            // init possible types of annotations
-            AnnotationInfo underlineAnnotation = InitAnnotationInfo();
-            // set line color
-            underlineAnnotation.PenColor = 1201033;
+            //SetFixTop(true);
+            underlineAnnotation = InitAnnotationBase(underlineAnnotation) as UnderlineAnnotation;
+            underlineAnnotation.FontColor = 1201033;
             return underlineAnnotation;
         }
-        
-        public override AnnotationInfo AnnotatePdf()
+
+        public override AnnotationBase AnnotatePdf()
         {
-            SetFixTop(false);
-            AnnotationInfo underlineAnnotation = InitAnnotationInfo();
-            underlineAnnotation.Guid = annotationData.id.ToString();
-            underlineAnnotation.AnnotationPosition = new Point(annotationData.left, annotationData.top);
-            underlineAnnotation.PenColor = 1201033;
+            //SetFixTop(false);
+            underlineAnnotation = InitAnnotationBase(underlineAnnotation) as UnderlineAnnotation;
+            underlineAnnotation.FontColor = 1201033;
             return underlineAnnotation;
         }
-        
-        public override AnnotationInfo AnnotateCells()
+
+        public override AnnotationBase AnnotateCells()
         {
             throw new NotSupportedException(String.Format(Message, annotationData.type));
         }
-        
-        public override AnnotationInfo AnnotateSlides()
+
+        public override AnnotationBase AnnotateSlides()
         {
-            SetFixTop(true);
-            AnnotationInfo underlineAnnotation = InitAnnotationInfo();
-            underlineAnnotation.PenColor = 0;
-            underlineAnnotation.AnnotationPosition = new Point(annotationData.left, annotationData.top);
+            //SetFixTop(true);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)